search/requirements.txt

```diff
@@ -1,27 +1,45 @@
+astroid==2.3.3
 boto==2.49.0
 boto3==1.9.225
 botocore==1.12.225
 certifi==2019.3.9
 chardet==3.0.4
 Click==7.0
+cycler==0.10.0
+Cython==0.29.14
 docutils==0.15.2
 Flask==1.0.3
 Flask-Cors==3.0.8
+gensim==3.8.0
 idna==2.8
+isort==4.3.21
 itsdangerous==1.1.0
 Jinja2==2.10.1
 jmespath==0.9.4
 joblib==0.13.2
+kiwisolver==1.1.0
+langdetect==1.0.7
+lazy-object-proxy==1.4.3
 MarkupSafe==1.1.1
+matplotlib==3.0.3
+mccabe==0.6.1
+nltk==3.4.5
+nose==1.3.7
+numpy==1.17.2
 psycopg2==2.7.6.1
+pylint==2.4.4
+pyparsing==2.4.6
 python-dateutil==2.8.0
 python-dotenv==0.10.2
 requests==2.22.0
 s3transfer==0.2.1
+scikit-learn==0.21.3
+scipy==1.3.1
 six==1.12.0
+sklearn==0.0
 smart-open==1.8.4
+typed-ast==1.4.1
 urllib3==1.25.3
 Werkzeug==0.15.4
 wincertstore==0.2
-gensim==3.8.1
-waitress==1.4.3
```

Comment:
Waitress je potreben za serviranje microservicea na windowsu.  Lahko se ga odstrani, če to ni del programa in se potem pove kaj je potrebno spremenit v kodi. Ker pa je koda malo odvisna od tega ali zaganjaš na windowsu ali ane, bi jaz tole pustil notri.